src/components/ProgressStep/index.js

```diff
@@ -92,9 +100,8 @@ class StepItem extends Component {
                 <ButtonIcon
                     icon={this.getIcon()}
                     className={this.getButtonClassNames()}
-                    onClick={this.handleOnClick}>
-                    <AssistiveText text={this.getAssistiveText()} />
-                </ButtonIcon>
+                    onClick={this.handleOnClick} />
+                <AssistiveText text={this.getAssistiveText()} />
```

Comment:
you don't need to use an AssistiveText here since the ButtonIcon has one inside, so use the assistiveText prop of the ButtonIcon instead